fem/eltrans.cpp

```diff
@@ -200,24 +200,65 @@ void IsoparametricTransformation::Transform (const DenseMatrix &matrix,
    }
 }
 
+const IntegrationPoint IsoparametricTransformation::FindInitialIntegrationPoint(
+   const Vector& pt)
+{
+   IntegrationPoint xip;
+
+   const int geom = FElem->GetGeomType();
+   const IntegrationRule* ir = &(RefinedIntRules.Get(geom, FElem->GetOrder() ) );
```

Comment:
The quadratic case appears to still work if we reduce the order here from 2 to 1.
e.g.
```cpp
   const int refineOrder = FElem->GetOrder(); // 1 might be sufficient ?
   const IntegrationRule* ir = &(RefinedIntRules.Get(geom, refineOrder ) );
```
Here are the paths taken by the solver in this case:
![c-shaped-25-initialize-refinementlevel1](https://user-images.githubusercontent.com/5626552/31646212-59d84190-b2b5-11e7-828b-31839ee03ee4.png)

The order 2 refinement rules have a 4x4 grid, while the order-1 rules have a 2x2 grid of integration points.

Does it make sense to add a parameter to the ``IsoparametricTransformation`` class  (or the ``TransformBack`` function) to let the user control the order?